pkgs/applications/window-managers/tinywl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, wlroots, pkg-config
+, libxkbcommon, pixman, udev, wayland, wayland-protocols
+}:
+
+stdenv.mkDerivation {
+  pname = "tinywl";
+  inherit (wlroots) version src;
+
+  sourceRoot = "source/tinywl";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libxkbcommon pixman udev wayland wayland-protocols wlroots ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp tinywl $out/bin
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/swaywm/wlroots/tree/master/tinywl";
+    description = ''"minimum viable product" Wayland compositor based on wlroots.'';
```

Comment:
Nitpick: The description shouldn't end with a period.
Maybe ``''A "minimum viable product" Wayland compositor based on wlroots''`` (prefixed with "A " it's capitalized and looks a bit less strange IMO)?

Reply:
Ah yes I agree.  I thought we had a rule against starting with an article but it appears I was mistaken.